web/profiles/custom/os2loop/themes/os2loop_theme/assets/components/pdf/pdf.scss

```diff
@@ -60,40 +116,44 @@ img {
 
 	.title-icon {
 		margin-right: 1em;
+        height: 30px;
+        width: 20px;
```

Comment:
Be consistent when indenting code, i.e. match the style in the rest of the file.